src/components/Cards/card-data/terra-station-cards.js

```diff
@@ -1,66 +1,59 @@
 const terraStationCards = [
   {
-    href: './download/',
+    href: '/learn/terra-station/download/',
     lightSvg: '/img/icons/station/station.svg',
     darkSvg: '/img/icons/dark/station.svg',
     header: 'Download Station',
     summary: 'Download Terra Station, the official wallet for Terra.'
   },
   {
-    href: './wallet.html',
+    href: '/learn/terra-station/wallet.html',
     lightSvg: '/img/icons/station/wallet.svg',
     darkSvg: '/img/icons/dark/wallet.svg',
     header: 'Create a wallet',
     summary: 'Create and manage a wallet.'
   },
   {
-    href: './networks.html',
+    href: 'https://station.terra.money/',
+    lightSvg: '/img/icons/Terrad.svg',
+    darkSvg: '/img/icons/dark/Terrad.svg',
+    header: 'Station WebApp',
+    summary: 'Utilize the Terra Station web application for a more complete experience.'
```

Comment:
I think we want to push people to download Station desktop more than use the WebApp? 

Unsure.

Reply:
@evanorti, think we should get rid of that card altogether?

Reply:
Yes. I think so